src/overlays/actors/ovl_En_Ishi/z_en_ishi.c

```diff
@@ -114,12 +113,14 @@ static ColliderCylinderInit sCylinderInit[] = {
 
 static CollisionCheckInfoInit sColChkInfoInit = { 0, 12, 60, MASS_IMMOVABLE };
 
-static s16 D_8095F74C[] = { 16, 13, 11, 9, 7, 5 };
+static s16 sIshiSmallRockDebrisScales[] = { 16, 13, 11, 9, 7, 5 };
 
-static s16 D_8095F758[] = { 145, 135, 120, 100, 70, 50, 45, 40, 35 };
+static s16 sIshiBoulderDebrisScales[] = { 145, 135, 120, 100, 70, 50, 45, 40, 35 };
 
-static s16 D_8095F76C[] = { -1, 1, 2, 20, 8, 0 };
+static s16 sIshiItemDrops[] = { ITEM00_NO_DROP,      ITEM00_RUPEE_BLUE, ITEM00_RUPEE_RED,
+                                ITEM00_RUPEE_PURPLE, ITEM00_ARROWS_30,  ITEM00_RUPEE_GREEN };
```

Comment:
```suggestion
static s16 sIshiItemDrops[] = { ITEM00_NO_DROP,      ITEM00_RUPEE_BLUE, ITEM00_RUPEE_RED,
                                ITEM00_RUPEE_PURPLE, ITEM00_ARROWS_30,  ITEM00_RUPEE_GREEN, };
```
Think this would format a little better.